docs/UserGuide.md

```diff
@@ -137,51 +121,15 @@ Placeholders are words in UPPER_CASE to show you what parameters you can supply
 
 ## Troubleshooting
 
-Something goes here...
+{% include_relative _ug/Troubleshooting.md %}
 
 ## FAQ
 
 {% include_relative _ug/FAQ.md %}
 
-## Future Extensions
-
-(NOT COMPLETED)
-
-1. **Food expiring soon / Date food bought**
-
-1. **Glorified search and sort**
-
-   1. Upgrade sort and search
-   1. Sort food items by quantity
-   1. Sort food items by name
-   1. Sort food items by expiry date
-   1. Sort food items by purchase date
-   {: .lower-alpha}
-
-1. **Food buffer**
-
-   1. Rainbow UI / Dashboard
-   1. Optional : Minimum acceptable quantity
-   1. Optional : Percentage of stock expiring
-   {: .lower-alpha}
-
-1. **Purchasing (Hard -> Will not see benefit immediately)**
-
-   1. History + Statistics
-   1. Inventory need a price of items
-   {: .lower-alpha}
-
-1. **(Last priority) Order management**
-
-   1. Grouping of items
-   1. Creation of menu with specific items
-   1. Record menu items bought
-   1. Statistics
-   {: .lower-alpha}
-
```

Comment:
Are we going to move this anywhere?

Reply:
We won't be using these future extensions (if we choose to write about future extensions) so they're find to remove

Reply:
Noted, thanks! Will mark this as resolved.